projects/admin/src/app/record/brief-view/statistics-cfg-brief-view-component.ts

```diff
@@ -29,8 +29,7 @@ import { ResultItem } from '@rero/ng-core';
     ></i>
   <a [routerLink]="[detailUrl.link]">{{ record.metadata.name }}</a></h5>
   <div class="card-text">
-    <div *ngIf="record.metadata.description">
-      {{ record.metadata.description }}
+    <div *ngIf="record.metadata.description" [innerHTML]="record.metadata.description | truncateText: 30">
```

Comment:
really 30 characters ? It seems very low.

Reply:
It is 30 words. It surprised me to have words and not characters.